src/client/pages/Checkout/CheckoutPayment/CheckoutPayment.tsx

```diff
@@ -465,8 +450,8 @@ export const CheckoutPayment = ({
 
     if (isFormDataUpdated) {
       sendDatadogAction('holder_form_submit')
-      const { data } = await submitForm()
-      const isUpdateQuotesFailed = isQuoteBundleError(data)
+      await submitForm()
+      const isUpdateQuotesFailed = isQuoteBundleError(editQuoteData)
```

Comment:
I personally prefer the previous setup.

It's easier to follow that we use the updated data we get from "submitForm" instead of a variable from outside the scope of this function.

Just a comment tho :)

Reply:
That's true. But then I'd need to return an array of `data` - one for each `editQuote` operation and check for at least one failed operation. But that would be fine I guess.

Reply:
Ahaa, didn't think of that.. Hmm perhaps not worth it then